src/amo/reducers/versions.js

```diff
@@ -106,7 +128,19 @@ export const getVersionsBySlug = ({
   invariant(state, 'state is required');
 
   const infoForSlug = state.bySlug[slug];
-  return (infoForSlug && infoForSlug.versions) || null;
+  const versionIds = infoForSlug && infoForSlug.versionIds;
```

Comment:
Why is this checking for the presence of `infoForSlug.versionIds` ? The [type](https://github.com/mozilla/addons-frontend/blob/7200d59388557d36d35dbede5b19d492af794e5c/src/amo/reducers/versions.js#L45) says it will always be an array. Does the type need updating?

Reply:
The code was not in sync with the Flow definition, but I fixed that. It now should always be an array, so I can remove the extra check.